src/lib/isMobilePhone.js

```diff
@@ -52,7 +52,7 @@ const phones = {
   'en-ZA': /^(\+?27|0)\d{9}$/,
   'en-ZM': /^(\+?26)?09[567]\d{7}$/,
   'en-ZW': /^(\+263)[0-9]{9}$/,
-  'es-AR': /^\+?549(11|[2368]\d)\d{8}$/,
+  'es-AR': /^\+?549?(11|[2368]\d)\d{8}$/,
```

Comment:
I am still in debate on this change. 
Since we are looking only at international inbound messages only for the change, I think it could be better not to make the change. If the change is necessary, then, adding it has an option could be better.
I am not sure what @tux-tn, @rubiin, or @profnandaa thinks about this change.

Reply:
I think we should focus on the wider application too

Reply:
Both are valid international numbers (with and without that "9")
But I get it if the goal is to favor a single format
Extra app logic would be required anyways before processing these numbers